src/components/pages/rules/rulesGrid/rulesGrid.js

```diff
@@ -23,7 +23,7 @@ export class RulesGrid extends Component {
 
     // TODO: This is a temporary example implementation. Remove with a better version
     this.contextBtns = {
-      disable: <Btn key="disable">Disable</Btn>,
+      disable: <Btn key="disable" onClick={() => console.log('Disabled!')}>Disable</Btn>,
```

Comment:
Remove the console log. And Move `Disable` to translation?

Reply:
Woops, this was to show an example to someone. Not translating atm since this was helper scaffolding code.